src/main/java/org/synyx/urlaubsverwaltung/person/extension/PersonServiceExtensionImpl.java

```diff
@@ -0,0 +1,95 @@
+package org.synyx.urlaubsverwaltung.person.extension;
+
+import de.focus_shift.urlaubsverwaltung.extension.api.person.PersonDTO;
+import de.focus_shift.urlaubsverwaltung.extension.api.person.PersonServiceExtension;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.stereotype.Service;
+import org.synyx.urlaubsverwaltung.person.Person;
+import org.synyx.urlaubsverwaltung.person.PersonService;
+import org.synyx.urlaubsverwaltung.search.PageStreamSupport;
+import org.synyx.urlaubsverwaltung.search.PageableSearchQuery;
+
+import java.util.Optional;
+import java.util.stream.Stream;
+
+import static org.synyx.urlaubsverwaltung.person.extension.PersonDTOMapper.toPerson;
+import static org.synyx.urlaubsverwaltung.person.extension.PersonDTOMapper.toPersonDTO;
+
+@ConditionalOnProperty(value = "uv.extensions.enabled", havingValue = "true")
+@Service
+public class PersonServiceExtensionImpl implements PersonServiceExtension {
```

Comment:
Benötigen wir in der extension alle interfaces oder würde es für den Start reichen nur die die wir brauchen zu implementieren?

Reply:
Es gibt im Extension-Service nur dieses Interface, also reicht es so aus, wie es aktuell ist. Nach und nach kann Extension-Service dann neue Interfaces anbieten, welche dann entsprechend von der UV implementiert werden.

D.h. es reicht fürs erste aus, mit Person* zu starten

Reply:
sry, nicht gut geschrieben, ich meinte die einzelnen Methoden.